src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
+export default class calculator extends Component {
+  render() {
```

Comment:
- [ ] Please use best practice, which in this case means the class name should be capitalized.